projects/packages/forms/src/blocks/contact-form/components/jetpack-email-connection-settings.js

```diff
@@ -87,44 +87,51 @@ const JetpackEmailConnectionSettings = ( {
 
 	return (
 		<>
-			<InspectorHint>
-				{ __( 'Get incoming form responses sent to your email inbox:', 'jetpack-forms' ) }
-			</InspectorHint>
-			<TextControl
-				aria-describedby={ `contact-form-${ instanceId }-email-${
-					hasEmailErrors() ? 'error' : 'help'
-				}` }
-				label={ __( 'Email address to send to', 'jetpack-forms' ) }
-				placeholder={ __( 'name@example.com', 'jetpack-forms' ) }
-				onKeyDown={ e => {
-					if ( event.key === 'Enter' ) {
-						e.preventDefault();
-						e.stopPropagation();
-					}
+			<ToggleControl
+				checked={ !! sendEmailOnSubmission }
+				label={ __( 'Enable email notifications', 'jetpack-forms' ) }
```

Comment:
I don't have a good alternative to offer now, but this copy can be improved; "Enable" is implicit from the toggle so something like "Send form submissions as email", or something along those lines.